test.js

```diff
@@ -48,8 +61,8 @@ test('should return an html document', t => {
   })
 })
 
-test('should support different methods', t => {
-  t.plan(3)
+test('static pages only support GET and HEAD requests', t => {
+  t.plan(9)
```

Comment:
shall we take the chance of making  this test async and remove the need to .plan()?